commands/account/auth.ts

```diff
@@ -102,9 +103,14 @@ type AccountAuthArgs = CommonArgs &
 export async function handler(
   args: ArgumentsCamelCase<AccountAuthArgs>
 ): Promise<void> {
-  const { providedAccountId, disableTracking } = args;
+  const { providedAccountId, disableTracking, config } = args;
   const authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value;
 
+  if (config) {
```

Comment:
I should have been a little clearer when I mentioned the existing config thing. I was referring to the case where the user already has a `hubspot.config.yml` file, and then tries to run `hs account auth`. In that case we should log an error or some sort of message to explain what they can do to port their existing config to the new format.

This `--config` case is still useful to catch, but I think you can solve this one by just removing `addConfigOptions()` in the builder. Then this command won't support `--config` at all.